types/nightwatch/test/testElementCommands.ts

```diff
@@ -0,0 +1,525 @@
+import { NightwatchAPI, JSON_WEB_OBJECT, NightwatchCallbackResult, NightwatchSizeAndPosition } from "nightwatch";
+
+function isNightwatchAPI(v: NightwatchAPI) {}
+function isNightwatchCallbackResult<T>(v1: NightwatchCallbackResult<T>, v2: NightwatchCallbackResult<T>) {}
+function isResultType<T>(v1: T, v2: T) {}
+
+//
+// .clearValue
+//
+describe('clearValue Command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .clearValue('input[type=text]', function(result) {
+                isNightwatchAPI(this);
+                isNightwatchCallbackResult<null>(result, result);
+            }).expect.element('input[type=text]').text.to.equal('');
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .clearValue('input[type=text]');
+        isResultType<null>(result, result);
+    });
+});
+
+//
+// .click
+//
+describe('click Command demo', function() {
+    test('demo test', function() {
+        browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .click('input[type=submit]', function(result) {
+                isNightwatchAPI(this);
+                isNightwatchCallbackResult<null>(result, result);
+            });
+    });
+
+    test('async demo test', async function(browser) {
+        const result = await browser
+            .url('https://google.com')
+            .waitForElementVisible('input[type=text]')
+            .setValue('input[type=text]', 'nightwatch.js')
+            .click('input[type=submit]');
+        isResultType(result, result);
```

Comment:
This check is missing the type. 
Maybe mark the generic type `T` in both `isResultType` and `isNightwatchCallbackResult` as `never`. This will always complain that the type of `result` is compatible with `never`. 
```ts
function isNightwatchCallbackResult<T extends never>(v1: NightwatchCallbackResult<T>, v2: NightwatchCallbackResult<T>) {}
function isResultType<T extends never>(v1: T, v2: T) {}
```